main.go

```diff
@@ -2,31 +2,50 @@ package main
 
 import (
 	"fmt"
-	"log"
-	"os"
-	"runtime/pprof"
-
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 	"github.com/uyuni-project/inter-server-sync/cli"
 	"github.com/uyuni-project/inter-server-sync/entityDumper"
 	"github.com/uyuni-project/inter-server-sync/schemareader"
+	"os"
+	"runtime/pprof"
 )
 
-func init() { log.SetFlags(log.Lshortfile | log.LstdFlags) }
+func loginit() {
+	Logfile := "/tmp/uyuni_iss_log.json"
```

Comment:
Should this file location be read from the parameters and fall back to this location in case it's not defined?
Or another option can be, do not write to file log, if it's not  defined in the command parameters.